packages/mdc-chips/README.md

```diff
@@ -43,20 +44,33 @@ npm install @material/chips
   <div class="mdc-chip" tabindex="0">
     <div class="mdc-chip__text">Chip content</div>
   </div>
-  <div class="mdc-chip" tabindex="0">
-    <div class="mdc-chip__text">Chip content</div>
-  </div>
-  <div class="mdc-chip" tabindex="0">
-    <div class="mdc-chip__text">Chip content</div>
-  </div>
+  ...
 </div>
 ```
 
-#### Leading and Trailing Icons
+### Styles
+
+```scss
+@import "@material/chips/mdc-chips";
+```
+
+### JavaScript Instantiation
+
+```js
+import {MDCChipSet} from '@material/chips';
+
+const chipSet = new MDCChipSet(document.querySelector('.mdc-chip-set'));
+```
+
+> See [Importing the JS component](../../docs/importing-js.md) for more information on how to import JavaScript.
+
+## Variants
+
+### Leading and Trailing Icons
```

Comment:
IMO I'd be inclined to put the different chip types first to quickly follow the example above, rather than require people reading this top-to-bottom to context-switch back later.

Also, do/should we explain anywhere which of leading/trailing are valid on which types of chip?

Reply:
I was thinking about that too, but Filter Chips references leading icons...

Reply:
I think it's better to add the note about input chips under trailing icons and have it linked to the input chips section.
Now it's debatable whether we should do the same for filter chips...but I think filter chips talks about leading icons in a depth that requires more than just a reference.